ttnn/cpp/ttnn/operations/eltwise/ternary/where_op.cpp

```diff
@@ -45,46 +45,46 @@ inline Tensor _where_op(
 }
 
 
-Tensor WhereOp::_where(
+Tensor ExecuteWhereOp::operator()(
```

Comment:
We're witching to WhereOperation, and WhereDeviceOperation respectively now

Reply:
Sorry i didn't get you @sjameelTT

Reply:
Name of the struct. We're moving to using WhereOperation instead of ExecuteWhereOp for example, last I heard. @ayerofieiev-tt  is this true or am I misinformed?

Reply:
Thats accurate. It is more related to following the latest core infra from @arakhmati which is based on templates. 
It is not required, but nice to have as it reduces future work required to remake

Reply:
Ok. will switch to WhereOperation